### DISCUSSION TITLE
깃헙으로 간편 로그인 구현하려고 하는데 어려움이 있어 질문드립니다.

### DISCUSSION THREAD

**Discussion by mkwkw** [devrel-kr/HackaLearn#134] [7 Q&A - 백엔드] [answered]
안녕하세요.
저희가 웹사이트 만드는 중인데요. 깃헙으로 간편 로그인하는 기능을 구현하려고합니다. 클라우드 스킬 챌린지 사용자 인증부분에서 나온대로 azure와 github으로 로그인을 연결하면 azure자체에서 깃헙으로 간편 로그인하는 것을 관리해주는 것인지, 따로 토큰을 받아와서 간편 로그인을 진행해야하는 것인지 궁금합니다. 
만약에 따로 토큰을 받아와야한다면 어떻게 구현하면 좋을지 알려주시기를 부탁드립니다.

감사합니다!

**Comment by justinyoo** [comment, marked as answer]
클라우드 스킬 챌린지에서 했던 것 처럼 하면 별다른 설정 없이 깃헙 ID로 로그인이 가능합니다. 혹시 로그인 후 JWT 같은 걸 활용하고자 하시는 건가요? 로그인 후 사용자 정보에 접근하려면 [이 페이지](https://docs.microsoft.com/ko-kr/azure/static-web-apps/user-information?tabs=javascript&WT.mc_id=devcloud-33673-juyoo&ocid=AID3035096)를 참조해 보세요.

**Reply by mkwkw** [reply]
감사합니다!

**Reply by minhoryang** [reply]
네 맞습니다. 그대로 사용하시면 문제 없을 것 같습니다.
